lib/r5.js

```diff
@@ -1,13 +1,10 @@
 const fetch = require('isomorphic-fetch')
-const tzwhere = require('tzwhere')
 
 const config = require('./config')
 
-// tzwhere.init()
-
 module.exports.requestPlan = function (plan) {
   if (config.r5) {
-    const timezoneOffset = -4 // millisecondsToHours(tzwhere.tzOffsetAt(plan.from.lat, plan.from.lng))
```

Comment:
May I recommed the use of a better timezone geolocation library?  Namely https://github.com/evansiroky/node-geo-tz/